### ISSUE TITLE
Add new filter ssl-cert or ssl cert name

### ISSUE THREAD

**Issue by mariudelros** [cea-sec/ivre#417]
Good day,
is it possible to know how may a query with IVRE (using filter), a scanning result done with masscan, about SSL cert banners? For example:

```
ssl-cert
MICRTCCAAaMVMBfgMwEQYJYIZytIAYsebAaMVMBMwjujEQYJYIZIAYrrhbAaMhBMwEQYJYIZIAYbAaMVMfBMwEQYJYIZIAdYbAaMVMBMwEQYJYIZIAYb....
```

OR

```
ssl
TLS/1.1 cipher:0x35, somewebsite.com
```

using the keyword banners or sshkey doesn't work.

Thank a lot. Nice tool.
Mariu

VERSION:

Dependencies:
    Python module Crypto: 2.6.1
    Python module pymongo: 2.7.2
    Python module py2neo: 3.1.2
    Python module sqlalchemy: 1.0.12
    Python module psycopg2: 2.7.3.1 (dt dec pq3 ext lo64)
    Python module krbV: missing
    Python module PIL: [unknown version]
    Python module MySQLdb: 1.2.3
    Python module dbus: 1.2.0
    Python module matplotlib: 1.3.1

**Comment by p-l-** [comment]
That's not possible (for now). I'm leaving this issue opened as a feature request, to track the task progress here.

**Comment by p-l-** [comment]
Fixed in #536. You will have to import your scan result again, after having upgraded to current master (at least version 0.9.10.dev82). Thanks a lot for reporting this.
